metadata/src/main/java/edu/unc/lib/dl/util/JobStatusFactory.java

```diff
@@ -198,18 +186,14 @@ public String getJobByStatus(String depositUUID, JobStatus status) {
 					return jobUUID;
 				}
 			}
-		} finally {
-			getJedisPool().returnResource(jedis);
 		}
 
 		return null;
 	}
 
 	public List<String> getJobsByStatus(String depositUUID, JobStatus status) {
-		Jedis jedis = getJedisPool().getResource();
-
-		List<String> result = new ArrayList<String>();
-		try {
+		List<String> result = new ArrayList<>();
```

Comment:
Just curious about why you reverted to using the raw ArrayList type here. I'm sure there's a good reason, :-)